src/NuGet.Core/NuGet.Common/Errors/NuGetLogCode.cs

```diff
@@ -857,5 +857,10 @@ public enum NuGetLogCode
         /// Undefined package warning
         /// </summary>
         NU5500 = 5500,
+
+        /// <summary>
+        /// InvalidUndottedFrameworkWarning
+        /// </summary>
+        NU5501 = 5501,
```

Comment:
We'll probably need a docs issue to add docs for this warning.

Reply:
https://github.com/NuGet/docs.microsoft.com-nuget/issues/2177